app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsActivityPresenter.kt

```diff
@@ -54,4 +54,8 @@ class DeveloperOptionsActivityPresenter @Inject constructor(
         R.id.developer_options_fragment_placeholder
       ) as DeveloperOptionsFragment?
   }
+
+  fun forceCrash() {
```

Comment:
All public functions should have KDocs explaining what they are doing & how/when they should/could be called. Please make sure all public functions, classes, and fields in this & your other PRs have clear KDocs.

Reply:
KDoc added.